sycl/doc/syclgraph/SYCLGraphUsageGuide.md

```diff
@@ -0,0 +1,515 @@
+# SYCL Graph Usage Guide and Examples
+
+This document describes recommended usage guidelines for using the
+`sycl_ext_oneapi_graph` extension (referred to as SYCL Graph in this document)
+as well as provides example code snippets for various features and usage
+scenarios.
+
+The specification for the `sycl_ext_oneapi_graph` extension can be found
+[here](../extensions/experimental/sycl_ext_oneapi_graph.asciidoc).
+
+## General Usage Guidelines
+
+The following section provides some general usage guidelines when working
+with SYCL Graph or adapting an existing SYCL application to use SYCL Graph.
+Examples here will generally only use one graph creation API ("Explicit" or
+"Record & Replay") for simplicity, but can generally be applied to either API
+unless specifically noted.
+
+### Use Host-Tasks For Host Work
+
+SYCL Graph cannot capture work done on the host in application code which does
+not go through the SYCL runtime, including direct operations (for example kernel
+submissions) through a backend API (level-zero, cuda, etc). Any work to be done
+on the host should be captured within a `sycl::host-task` as a node in the
```

Comment:
```suggestion
on the host should be captured within a `sycl::host_task` as a node in the
```

or maybe `handler::host_task`

Reply:
I've removed the monospace from this usage and just gone with "host-task" to be consistent with other usage, and I don't think the monospace is adding anything here.